jenkins/ci.suse.de/ardana-job.yaml

```diff
@@ -44,6 +44,12 @@
           description: >-
             This name will become the build name of the job.
 
+      - string:
+          name: cloudsource
+          default: 'SUSE-OpenStack-Cloud-8-devel-staging'
+          description: >-
+            This is used as input repository (from provo-clouddata) for testing
+
```

Comment:
Isn't ```deployer_model``` missing as parameter here?

Reply:
There is a parameter called "model" on line 35 which was added as part of 88525d5a77